src/registry/export_sdl.rs

```diff
@@ -207,8 +217,13 @@ impl Registry {
                     }
                     write!(sdl, "scalar {} ", name).ok();
 
-                    if options.federation && *inaccessible {
-                        write!(sdl, "@inaccessible ").ok();
+                    if options.federation {
+                        if *inaccessible {
+                            write!(sdl, "@inaccessible ").ok();
```

Comment:
btw this other MR https://github.com/async-graphql/async-graphql/pull/1036 wants to change this line (and the write a few lines above) to move the space to the front.